sql/mozfun/utils/get_url_host/udf.sql

```diff
@@ -0,0 +1,16 @@
+CREATE OR REPLACE FUNCTION utils.get_url_host(url STRING)
```

Comment:
I believe [NET.HOST(url)](https://cloud.google.com/bigquery/docs/reference/standard-sql/net_functions#nethost) can achieve this already :)

Reply:
woooooow thank you that is way easier \o/